src/platform/site-wide/user-nav/containers/Main.jsx

```diff
@@ -129,12 +129,13 @@ export class Main extends React.Component {
   };
 
   signInSignUp = () => {
-    const { formAutoSavedStatus } = this.props;
+    const { formAutoSavedStatus, additionalRoutes = [] } = this.props;
```

Comment:
With these two form-related props being only used here in one place, I wonder if it would be better to derive the `shouldConfirmLeavingForm` prop from them directly in the `mapStateToProps`.